pkg/loop/reporting_plugins/loopp_service_test.go

```diff
@@ -0,0 +1,81 @@
+package reporting_plugins_test
+
+import (
+	"os/exec"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/test"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/reporting_plugins"
+	"github.com/smartcontractkit/chainlink-relay/pkg/types"
+	utilstests "github.com/smartcontractkit/chainlink-relay/pkg/utils/tests"
+)
+
+func HelperProcess(command string, opts test.HelperProcessOptions) *exec.Cmd {
+	return test.HelperProcess("../internal/test/cmd/main.go", command, opts)
+}
+
+// pluginGenericMedian is a generic plugin which wants a median provider
+const pluginGenericMedian = "generic-median"
```

Comment:
the value `generic-median` is used in on of the tests in a switch. should we export this and update the test?